packages/block-editor/src/components/block-list/style.scss

```diff
@@ -182,7 +182,7 @@
 	.block-editor-block-list__block.is-multi-selected {
 		// Show selection borders around every non-nested block's actual footprint.
 		> .block-editor-block-list__block-edit > [data-block] {
-			box-shadow: 0 0 0 2px #057cba;
+			box-shadow: 0 0 0 2px $blue-blueberry, 0 0 0 4px $white;
```

Comment:
Love this. Guaranteed to see something.

Reply:
I played a bit with a dashed line (white and blue), but it's a bit too busy. Perhaps if the dashed lines could be longer, it would work better (maybe border image). In any case this is looking good.